framework7/template7.d.ts

```diff
@@ -0,0 +1,24 @@
+// Type definitions for Template7
```

Comment:
This should go in a `template7` package. (`npm run new-package template7`) Same for `dom7`.